browser/ui/webui/brave_web_ui_controller_factory.cc

```diff
@@ -99,13 +98,6 @@ WebUIController* NewWebUI(WebUI* web_ui, const GURL& url) {
         *profile->GetPrefs());
   } else if (host == kSkusInternalsHost) {
     return new SkusInternalsUI(web_ui, url.host());
-#if !BUILDFLAG(IS_ANDROID)
```

Comment:
Why this is removed? Looks like android uses AndroidWalletPageUI instead?

Reply:
AndroidWalletPageUI is for android and is not touched now. For desktop we now use WalletPageUIConfig to setup web ui